src/access/acp.test.ts

```diff
@@ -1032,6 +1056,73 @@ describe("getActorAccess", () => {
     });
   });
 
+  it("applies a Policy that also specifies empty Rules", () => {
+    const resourceWithAcr = mockResourceWithAcr(
+      "https://some.pod/resource",
+      "https://some.pod/resource?ext=acr",
+      {
+        policies: {
+          "https://some.pod/resource?ext=acr#policy": {
+            allow: { read: true },
+            allOf: {
+              "https://some.pod/resource?ext=acr#emptyRule": {},
+            },
+          },
+        },
+        memberPolicies: {},
+        acrPolicies: {},
+        memberAcrPolicies: {},
+      }
+    );
+
+    const access = internal_getActorAccess(resourceWithAcr, acp.agent, webId);
+
+    expect(access).toStrictEqual({
+      read: true,
+    });
+  });
```

Comment:
I'm not sure that's what I understand from the latest discussions : 
> upon evaluation a policy will only be considered if it has at least one valid rule. And a rule is only considered valid if it has at least one valid triple aside from its type triple.

So in this case, shouldn't the previous behaviour prevail, and the policy not apply for its lack of valid rules ?

Reply:
I added this when Emmet said that the behaviour implemented here is what should be needed. After that the discussion was re-opened and hasn't yet reached a conclusion, so when it does we might indeed need to revisit this. Unfortunately the discussions were in a private chat channel (because they're still preparing it for the official proposal) so I can't link them here, but I'll send the links to you.